from0k2bp.tex

```diff
@@ -959,85 +977,81 @@
 \subsection[Aside: the Sigma
 protocol]{\texorpdfstring{\protect\hypertarget{anchor-35}{}{}Aside: the
 Sigma
 protocol}{Aside: the Sigma protocol}}\label{aside-the-sigma-protocol}}
 
-This is an abstraction, worth mentioning at this point, because we are
-about to see another example, and we have already seen two. Here they
-were:
-
-P → V: $R$ (a new random curve point, but P knows $k$ s.t. $R=kG$)
-
-V → P: $e$ (a random scalar)
-
-P → V: $s$ (which P calculated from the equation $s=k+ex$)
+This is an abstraction worth mentioning at this point, because we are
+about to see another example, and we have already seen two -- the first was 
+Schnorr's identity protocol and the second was the proof of knowledge 
+of a set of vectors. Here they were:
 
-and
-
-P → V: $C_0$ (a new commitment to a newly chosen random vector of dimension $N$)
-
-V → P: $e$ (a random scalar)
-
-P → V: $(\mathbf{z}, s)$ (a single vector of dimension $N$, and another scalar)
+\begin{itemize}
+\item $\mathcal{P}$ → $\mathcal{V}$: $R$ (a new random curve point, but $\mathcal{P}$ knows $k$ s.t. $R=kG$)
+\item $\mathcal{V}$ → $\mathcal{P}$: $e$ (a random scalar)
+\item $\mathcal{P}$ → $\mathcal{V}$: $s$ (which $\mathcal{P}$ calculated from the equation $s=k+ex$), and
+\item $\mathcal{P}$ → $\mathcal{V}$: $C_0$ (a new commitment to a newly chosen random vector of dimension $N$)
+\item $\mathcal{V}$ → $\mathcal{P}$: $e$ (a random scalar)
+\item $\mathcal{P}$ → $\mathcal{V}$: $(\mathbf{z}, s)$ (a single vector of dimension $N$, and another scalar). 
+\end{itemize}
 
-(the first was Schnorr's identity protocol; the second was the proof of
-knowledge of a set of vectors). These are both examples of Sigma
+These are both examples of Sigma
 protocols, so called because of a vague resemblance to the greek letter
 $\Sigma$, in that the process goes forwards once, then backwards, then forwards
 finally. The common pattern, though, is more than this three step
 interactive process. We generalise it as something like:
 
-P → V: commitment
-
-V → P: challenge
-
-P → V: response (proof)
+\begin{itemize}
+\item $\mathcal{P}$ → $\mathcal{V}$: commitment
+\item $\mathcal{V}$ → $\mathcal{P}$: challenge
+\item $\mathcal{P}$ → $\mathcal{V}$: response (proof)
+\end{itemize}
 
 It's worth emphasizing this structure, and why it arises, at this point
-in the discourse, so the mathematical structure of the inner product
-proof isn't too overwhelming. Consider the Schnorr case (basically the
+in the discourse. It makes the mathematical structure of the inner product
+proof less overwhelming then. Consider the Schnorr case (basically the
 simplest) to see why it's necessary for the provision of a proof. Say
-you have a secret value x.
+you have a secret value $x$.
 
-\underline{Option 1: just send $x$ to V}
+\underline{Option 1: just send $x$ to $\mathcal{V}$}
 
 This would be fine except we're trying to keep $x$ secret, so \ldots{}
 
-\underline{Option 2: send to V}
```

Comment:
OK, so there was another error here that neither you or I noticed before: `send to V` - I believe I originally had in mind `send k + x to V, with k random` (something like that). Point being you can send x in a blinded form but it's useless unless V can verify that what was blinded was the right x.

Reply:
I have filed the issue for this ((#12). I believe this PR is too big now to make it even bigger.